packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -133,6 +133,7 @@ class ReorderableList extends StatefulWidget {
     this.keyboardDismissBehavior = ScrollViewKeyboardDismissBehavior.manual,
     this.restorationId,
     this.clipBehavior = Clip.hardEdge,
+    this.autoScrollerVelocityScalar,
```

Comment:
Awesome! Can you also add this to the one in the material library?

Reply:
Done